seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/config/JdbcOptions.java

```diff
@@ -67,7 +67,7 @@ public interface JdbcOptions {
     Option<Integer> FETCH_SIZE =
             Options.key("fetch_size")
                     .intType()
-                    .defaultValue(0)
+                    .defaultValue(1024)
                     .withDescription(
                             "For queries that return a large number of objects, "
```

Comment:
Please do not change this value. `0` mean use default value of jdbc driver. Use `1024` as default value could affect some db read performance.